azure-pipelines.yml

```diff
@@ -0,0 +1,120 @@
+
+trigger:
+  - master
+
+variables:
+  MONO_VERSION: 5_18_1
+  XCODE_VERSION: 10.2.1
+  CAKE_NUGET_USEINPROCESSCLIENT: false
```

Comment:
There seems to be some issues all of a sudden... Using this as an interim solution.